src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.cs

```diff
@@ -98,11 +98,12 @@ internal void Setup()
             // First, acquire any services we need throughout our lifetime.
             this.GetServices();
 
-            var componentModel = this.Package.ComponentModel;
+            // TODO: Can the below be removed?
```

Comment:
comment is vague.  maybe make it clearer that you mean the statement directly underneath and how it appears to have no valuable runtime impact.